lib/pantry.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/ingredient'
+
+class Pantry
+  attr_reader :ingredients
+
+  def initialize
+    @ingredients = []
+  end
+
+  def stock
```

Comment:
This will work for the initial value of stock being empty {}, but then as we start to add things, this won't accumulate that data for us. We'd want to store @stock as an instance variable instead of @ingredients. @stock = {} and then you can access that with stock_check and restock to accumulate the data.